R/token-define.R

```diff
@@ -1,4 +1,4 @@
-token <- tribble(
+token <- tibble::tribble(
```

Comment:
Just realised that we import all of {tibble} in the {styler} namespace. Can you check if we can drop that import and use `::` everywhere (maybe we already do so)?

Reply:
Done.